cli/azd/docs/manual-pipeline-config.md

```diff
@@ -0,0 +1,116 @@
+
+- `fork and clone`: Similar to `git clone`, but in this case, two `git remote` are created. `origin` will be pointing to the forked repository and typically `upstream` will be pointing to the original repository.
+
+Depending on what approach you used to start your project, there's a few steps you need to follow before setting up the pipeline.
+
+### azd init -t path-to-git-repo
+
+This is usually the simplest way, as the git repository is only local. The next step is to decide where the source will be pushed, for example, GitHub or Azure DevOps. Go and manually create the repository you will use and then run the next command from your local project `git remote add origin path-to-the-repo`.
+
+You can verify the remote was set correctly running `git remote -v`.
+
+At this point, you can continue on [Set up remote repository](#set-up-remote-repository)
+
+### git clone path-to-git-repo
+
+There are at least two reasons why you might use `git clone`. 
+
+1. You have previously set up your GitHub or Azure Devops repository and you are trying to add `azd` support for it. In this case, you own the repo that you clone and you will be pushing your source updates to this repo. If this is the case, you remote `origin` is correctly set and you can move on to [Set up remote repository](#set-up-remote-repository).
+
+1. You found a repository that you want to try, for example, from [Azure-Samples](https://github.com/Azure-Samples) and you clone it locally. In this case, the remote `origin` is pointing to the original repository and you won't be able to push changes to this repo. You might want to submit PR changes to the repo, but if you are thinking on setting up a pipeline and your own repository, first you need to update the remote origin. One option is to remove the remote origin with `git remote remove origin`. This will disconnect the repo from where it was cloned and you will be in a state where you can follow the steps from [azd init -t path-to-git-repo](#azd-init--t-path-to-git-repo).
+
+### zip download
+
+After extracting the code, you will need to run `git init -b main` to create a new local git repository with an initial branch `main`. Then you can follow the steps from [azd init -t path-to-git-repo](#azd-init--t-path-to-git-repo).
+
+> Note: This approach can introduce some issue on Windows, as some file's information might be stored within the .git folder and be lost after the zip download (which does not include the .git folder). 
+
+### fork and clone
+
+A fork and clone usually means that you have created your own repository but you still want to track what happens on the original repo. If you are not really planning to keep an eye to the main repository, you can optionally remote the remote `upstream` (or whatever is the name for the main repo) and to only let the `origin` remote. Since your remote is correctly set, you can move on to [Set up remote repository](#set-up-remote-repository)
```

Comment:
```suggestion
A fork and clone usually means that you have created your own repository but you still want to track what happens on the original repo. If you are not really planning to keep an eye to the main repository, you can optionally remove the remote `upstream` (or whatever is the name for the main repo) leaving only the `origin` remote. Since your remote is correctly set, you can move on to [Set up remote repository](#set-up-remote-repository)
```